### ISSUE TITLE
Unable to launch instance using r3.large

### ISSUE THREAD

**Issue by kingsd041** [rancher/os#2307]
**RancherOS Version: (ros os version)**
Only v1.3.0-rc2
**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
AWS

Using rancherosv1.3.0-rc2 and r3.large failed to launch the instance

I can find the system log of the instance in aws
```
 in your root= kernel command line option
[    0.000000] e820: last_pfn = 0x3e0900 max_arch_pfn = 0x400000000
[    0.000000] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WC  UC- WT  
[    0.000000] e820: last_pfn = 0xf0000 max_arch_pfn = 0x400000000
[    0.000000] found SMP MP-table at [mem 0x000fbc20-0x000fbc2f] mapped at [ffff8f87400fbc20]
[    0.000000] Scanning 1 areas for low memory corruption
[    0.000000] RAMDISK: [mem 0x7c6b5000-0x7fffffff]
[    0.000000] ACPI: Early table checksum verification disabled
[    0.000000] ACPI: RSDP 0x00000000000EA020 000024 (v02 Xen   )
[    0.000000] ACPI: XSDT 0x00000000FC00DDC0 000054 (v01 Xen    HVM      00000000 HVML 00000000)
[    0.000000] ACPI: FACP 0x00000000FC00DA80 0000F4 (v04 Xen    HVM      00000000 HVML 00000000)
[    0.000000] ACPI: DSDT 0x00000000FC001CE0 00BD19 (v02 Xen    HVM      00000000 INTL 20090123)
[    0.000000] ACPI: FACS 0x00000000FC001CA0 000040
[    0.000000] ACPI: FACS 0x00000000FC001CA0 000040
[    0.000000] ACPI: APIC 0x00000000FC00DB80 0000D8 (v02 Xen    HVM      00000000 HVML 00000000)
[    0.000000] ACPI: HPET 0x00000000FC00DCD0 000038 (v01 Xen    HVM      00000000 HVML 00000000)
[    0.000000] ACPI: WAET 0x00000000FC00DD10 000028 (v01 Xen    HVM      00000000 HVML 00000000)
[    0.000000] ACPI: SSDT 0x00000000FC00DD40 000031 (v02 Xen    HVM      00000000 INTL 20090123)
[    0.000000] ACPI: SSDT 0x00000000FC00DD80 000031 (v02 Xen    HVM      00000000 INTL 20090123)
[    0.000000] No NUMA configuration found
[    0.000000] Faking a node at [mem 0x0000000000000000-0x00000003e08fffff]
[    0.000000] NODE_DATA(0) allocated [mem 0x3e08fb000-0x3e08fffff]
[    0.000000] Zone ranges:
[    0.000000]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.000000]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.000000]   Normal   [mem 0x0000000100000000-0x00000003e08fffff]
[    0.000000]   Device   empty
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000000001000-0x000000000009dfff]
[    0.000000]   node   0: [mem 0x0000000000100000-0x00000000efffffff]
[    0.000000]   node   0: [mem 0x0000000100000000-0x00000003e08fffff]
[    0.000000] Initmem setup node 0 [mem 0x0000000000001000-0x00000003e08fffff]
[    0.000000] ACPI: PM-Timer IO Port: 0xb008
[    0.000000] IOAPIC[0]: apic_id 1, version 17, address 0xfec00000, GSI 0-47
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 low level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 low level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 low level)
[    0.000000] Using ACPI (MADT) for SMP configuration information
[    0.000000] ACPI: HPET id: 0x8086a201 base: 0xfed00000
[    0.000000] smpboot: Allowing 15 CPUs, 13 hotplug CPUs
[    0.000000] PM: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.000000] PM: Registered nosave memory: [mem 0x0009e000-0x0009ffff]
[    0.000000] PM: Registered nosave memory: [mem 0x000a0000-0x000dffff]
[    0.000000] PM: Registered nosave memory: [mem 0x000e0000-0x000fffff]
[    0.000000] PM: Registered nosave memory: [mem 0xf0000000-0xfbffffff]
[    0.000000] PM: Registered nosave memory: [mem 0xfc000000-0xffffffff]
[    0.000000] e820: [mem 0xf0000000-0xfbffffff] available for PCI devices
[    0.000000] Booting paravirtualized kernel on Xen HVM
[    0.000000] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645519600211568 ns
[    0.000000] setup_percpu: NR_CPUS:256 nr_cpumask_bits:256 nr_cpu_ids:15 nr_node_ids:1
[    0.000000] percpu: Embedded 36 pages/cpu @ffff8f8b20200000 s109656 r8192 d29608 u262144
[    0.000000] PV qspinlock hash table entries: 256 (order: 0, 4096 bytes)
[    0.000000] Built 1 zonelists in Node order, mobility grouping on.  Total pages: 3937380
[    0.000000] Policy zone: Normal
[    0.000000] Kernel command line: BOOT_IMAGE=../vmlinuz-4.9.80-rancher printk.devkmsg=on rancher.state.dev=LABEL=RANCHER_STATE rancher.state.wait panic=10 console=ttyS0 rancher.cloud_init.datasources=["ec2"] initrd=../initrd-v1.3.0-rc2
[    0.000000] PID hash table entries: 4096 (order: 3, 32768 bytes)
[    0.000000] Memory: 15604124K/15999604K available (7078K kernel code, 1473K rwdata, 3720K rodata, 1472K init, 1264K bss, 395480K reserved, 0K cma-reserved)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=15, Nodes=1
[    0.000000] Kernel/User page tables isolation: enabled
[    0.000000] Hierarchical RCU implementation.
[    0.000000] 	Build-time adjustment of leaf fanout to 64.
[    0.000000] 	RCU restricting CPUs from NR_CPUS=256 to nr_cpu_ids=15.
[    0.000000] RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=15
[    0.000000] NR_IRQS:16640 nr_irqs:952 16
[    0.000000] xen:events: Using 2-level ABI
[    0.000000] xen:events: Xen HVM callback vector for event delivery is enabled
[    0.000000] Console: colour VGA+ 80x25
[    0.000000] Cannot get hvm parameter CONSOLE_EVTCHN (18): -22!
[    0.000000] console [ttyS0] enabled
[    0.000000] clocksource: hpet: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 30580167144 ns
[    0.000000] tsc: Fast TSC calibration using PIT
[    0.000000] tsc: Detected 2500.048 MHz processor
[    0.020000] Calibrating delay loop (skipped), value calculated using timer frequency.. 5000.16 BogoMIPS (lpj=10000320)
[    0.024003] pid_max: default: 32768 minimum: 301
[    0.026005] ACPI: Core revision 20160831
[    0.033579] ACPI: 3 ACPI AML tables successfully acquired and loaded
[    0.036030] Security Framework initialized
[    0.037840] Yama: becoming mindful.
[    0.039386] SELinux:  Initializing.
[    0.040011] AppArmor: AppArmor disabled by boot time parameter
[    0.044915] Dentry cache hash table entries: 2097152 (order: 12, 16777216 bytes)
[    0.051446] Inode-cache hash table entries: 1048576 (order: 11, 8388608 bytes)
[    0.056244] Mount-cache hash table entries: 32768 (order: 6, 262144 bytes)
[    0.059227] Mountpoint-cache hash table entries: 32768 (order: 6, 262144 bytes)
[    0.060413] CPU: Physical Processor ID: 0
[    0.064019] mce: CPU supports 2 MCE banks
[    0.065848] Last level iTLB entries: 4KB 512, 2MB 8, 4MB 8
[    0.068003] Last level dTLB entries: 4KB 512, 2MB 0, 4MB 0, 1GB 4
[    0.070708] Spectre V2 mitigation: Mitigation: Full generic retpoline
[    0.073289] Freeing SMP alternatives memory: 20K
[    0.077430] ftrace: allocating 34218 entries in 134 pages
[    0.128085] smpboot: x86_max_cores == zero !?!?[    0.130067] smpboot: Max logical packages: 15
[    0.132620] x2apic: IRQ remapping doesn't support X2APIC mode
[    0.135698] Switched APIC routing to physical flat.
[    0.138060] ..TIMER: vector=0x30 apic1=0 pin1=2 apic2=0 pin2=0
[    0.182377] clocksource: xen: mask: 0xffffffffffffffff max_cycles: 0x1cd42e4dffb, max_idle_ns: 881590591483 ns
[    0.184013] installing Xen timer for CPU 0
[    0.188072] smpboot: CPU0: Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz (family: 0x6, model: 0x3e, stepping: 0x4)
[    0.192037] cpu 0 spinlock event irq 53
[    0.193824] Performance Events: unsupported p6 CPU model 62 no PMU driver, software events only.
[    0.198441] NMI watchdog: disabled (cpu0): hardware events not enabled
[    0.200005] NMI watchdog: Shutting down hard lockup detector on all cpus
[    0.203196] installing Xen timer for CPU 1
[    0.204061] x86: Booting SMP configuration:
[    0.206001] .... node  #0, CPUs:        #1[    0.210489] cpu 1 spinlock event irq 59
[    0.212006] x86: Booted up 1 node, 2 CPUs
[    0.213898] smpboot: Total of 2 processors activated (10000.32 BogoMIPS)
[    0.217018] devtmpfs: initialized
[    0.217642] x86/mm: Memory block size: 128MB
[    0.225026] evm: security.selinux
[    0.226621] evm: security.SMACK64
[    0.228005] evm: security.SMACK64EXEC
[    0.229710] evm: security.SMACK64TRANSMUTE
[    0.231597] evm: security.SMACK64MMAP
[    0.232005] evm: security.ima
[    0.233466] evm: security.capability
[    0.235160] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    0.236023] futex hash table entries: 4096 (order: 6, 262144 bytes)
[    0.239028] pinctrl core: initialized pinctrl subsystem
[    0.240122] RTC time: 13:46:10, date: 03/27/18
[    0.242311] NET: Registered protocol family 16
[    0.252008] cpuidle: using governor ladder
[    0.264007] cpuidle: using governor menu
[    0.266006] PCCT header not found.
[    0.267763] ACPI: bus type PCI registered
[    0.268011] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.271377] PCI: Using configuration type 1 for base access
[    0.288050] HugeTLB registered 2 MB page size, pre-allocated 0 pages
[    0.292107] ACPI: Added _OSI(Module Device)
[    0.292107] ACPI: Added _OSI(Processor Device)
[    0.294158] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.296005] ACPI: Added _OSI(Processor Aggregator Device)
[    0.302495] ACPI: Interpreter enabled
[    0.304019] ACPI: (supports S0 S3 S4 S5)
[    0.305929] ACPI: Using IOAPIC for interrupt routing
[    0.308025] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.364764] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.367727] acpi PNP0A03:00: _OSC: OS supports [ASPM ClockPM Segments MSI]
[    0.368012] acpi PNP0A03:00: _OSC failed (AE_NOT_FOUND); disabling ASPM
[    0.371045] acpi PNP0A03:00: fail to add MMCONFIG information, can't access extended PCI configuration space under this bridge.
[    0.373220] acpiphp: Slot [0] registered
[    0.375955] acpiphp: Slot [3] registered
[    0.376299] acpiphp: Slot [4] registered
[    0.378399] acpiphp: Slot [5] registered
[    0.380298] acpiphp: Slot [6] registered
[    0.382416] acpiphp: Slot [7] registered
[    0.384285] acpiphp: Slot [8] registered
[    0.386442] acpiphp: Slot [9] registered
[    0.388293] acpiphp: Slot [10] registered
[    0.390453] acpiphp: Slot [11] registered
[    0.392286] acpiphp: Slot [12] registered
[    0.394460] acpiphp: Slot [13] registered
[    0.396286] acpiphp: Slot [14] registered
[    0.398424] acpiphp: Slot [15] registered
[    0.400292] acpiphp: Slot [16] registered
[    0.402430] acpiphp: Slot [17] registered
[    0.404289] acpiphp: Slot [18] registered
[    0.406431] acpiphp: Slot [19] registered
[    0.408288] acpiphp: Slot [20] registered
[    0.410451] acpiphp: Slot [21] registered
[    0.412291] acpiphp: Slot [22] registered
[    0.414451] acpiphp: Slot [23] registered
[    0.416289] acpiphp: Slot [24] registered
[    0.418443] acpiphp: Slot [25] registered
[    0.420298] acpiphp: Slot [26] registered
[    0.422499] acpiphp: Slot [27] registered
[    0.424293] acpiphp: Slot [28] registered
[    0.426439] acpiphp: Slot [29] registered
[    0.428290] acpiphp: Slot [30] registered
[    0.430490] acpiphp: Slot [31] registered
[    0.432287] PCI host bridge to bus 0000:00
[    0.434184] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.436006] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.439105] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.440005] pci_bus 0000:00: root bus resource [mem 0xf0000000-0xfbffffff window]
[    0.443400] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.449617] pci 0000:00:01.1: legacy IDE quirk: reg 0x10: [io  0x01f0-0x01f7]
[    0.452006] pci 0000:00:01.1: legacy IDE quirk: reg 0x14: [io  0x03f6]
[    0.454990] pci 0000:00:01.1: legacy IDE quirk: reg 0x18: [io  0x0170-0x0177]
[    0.456005] pci 0000:00:01.1: legacy IDE quirk: reg 0x1c: [io  0x0376]
[    0.459723] * Found PM-Timer Bug on the chipset. Due to workarounds for a bug,
[    0.459723] * this clock source is slow. Consider trying other clock sources
[    0.461228] pci 0000:00:01.3: quirk: [io  0xb000-0xb03f] claimed by PIIX4 ACPI
[    0.473539] ACPI: PCI Interrupt Link [LNKA] (IRQs *5 10 11)
[    0.476635] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.479734] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.480985] ACPI: PCI Interrupt Link [LNKD] (IRQs *5 10 11)
[    0.501370] ACPI: Enabled 2 GPEs in block 00 to 0F
[    0.503865] xen:balloon: Initialising balloon driver
[    0.504048] random: fast init done
[    0.505726] xen_balloon: Initialising balloon driver
[    0.508136] vgaarb: setting as boot device: PCI:0000:00:02.0
[    0.510866] vgaarb: device added: PCI:0000:00:02.0,decodes=io+mem,owns=io+mem,locks=none
[    0.516006] vgaarb: loaded
[    0.517311] vgaarb: bridge control possible 0000:00:02.0
[    0.520000] SCSI subsystem initialized
[    0.520048] ACPI: bus type USB registered
[    0.524024] usbcore: registered new interface driver usbfs
[    0.526526] usbcore: registered new interface driver hub
[    0.528033] usbcore: registered new device driver usb
[    0.530441] PCI: Using ACPI for IRQ routing
[    0.532567] NetLabel: Initializing
[    0.534137] NetLabel:  domain hash size = 128
[    0.536005] NetLabel:  protocols = UNLABELED CIPSOv4
[    0.538317] NetLabel:  unlabeled traffic allowed by default
[    0.540104] HPET: 3 timers in total, 0 timers will be used for per-cpu timer
[    0.543289] hpet0: at MMIO 0xfed00000, IRQs 2, 8, 0
[    0.544637] hpet0: 3 comparators, 64-bit 62.500000 MHz counter
[    0.550042] clocksource: Switched to clocksource xen
[    0.562875] VFS: Disk quotas dquot_6.6.0
[    0.564856] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.569622] pnp: PnP ACPI init
[    0.571122] system 00:00: [mem 0x00000000-0x0009ffff] could not be reserved
[    0.574519] system 00:01: [io  0x08a0-0x08a3] has been reserved
[    0.577212] system 00:01: [io  0x0cc0-0x0ccf] has been reserved
[    0.579892] system 00:01: [io  0x04d0-0x04d1] has been reserved
[    0.582802] system 00:07: [io  0x10c0-0x1141] has been reserved
[    0.585547] system 00:07: [io  0xb044-0xb047] has been reserved
[    0.605653] pnp: PnP ACPI: found 8 devices
[    0.616142] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.620410] NET: Registered protocol family 2
[    0.622709] TCP established hash table entries: 131072 (order: 8, 1048576 bytes)
[    0.626356] TCP bind hash table entries: 65536 (order: 8, 1048576 bytes)
[    0.629573] TCP: Hash tables configured (established 131072 bind 65536)
[    0.632646] UDP hash table entries: 8192 (order: 6, 262144 bytes)
[    0.635480] UDP-Lite hash table entries: 8192 (order: 6, 262144 bytes)
[    0.638604] NET: Registered protocol family 1
[    0.640760] pci 0000:00:00.0: Limiting direct PCI/PCI transfers
[    0.643767] pci 0000:00:01.0: PIIX3: Enabling Passive Release
[    0.646667] pci 0000:00:01.0: Activating ISA DMA hang workarounds
[    0.649647] pci 0000:00:02.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    0.653632] Unpacking initramfs...
[   10.986620] Freeing initrd memory: 58668K
[   10.988533] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[   10.991250] software IO TLB [mem 0xec000000-0xf0000000] (64MB) mapped at [ffff8f882c000000-ffff8f882fffffff]
[   10.995548] RAPL PMU: API unit is 2^-32 Joules, 3 fixed counters, 163840 ms ovfl timer
[   10.998886] RAPL PMU: hw unit of domain pp0-core 2^-16 Joules
[   11.001282] RAPL PMU: hw unit of domain package 2^-16 Joules
[   11.003616] RAPL PMU: hw unit of domain dram 2^-16 Joules
[   11.006026] Scanning for low memory corruption every 60 seconds
[   11.009060] audit: initializing netlink subsys (disabled)
[   11.011395] audit: type=2000 audit(1522158381.454:1): initialized
[   11.014313] Initialise system trusted keyrings
[   11.016352] workingset: timestamp_bits=40 max_order=22 bucket_order=0
[   11.020634] zbud: loaded
[   11.022257] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[   11.024995] fuse init (API version 7.26)
[   11.028237] Key type asymmetric registered
[   11.029986] Asymmetric key parser 'x509' registered
[   11.032116] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 248)
[   11.035238] io scheduler noop registered
[   11.036964] io scheduler deadline registered
[   11.038745] io scheduler cfq registered (default)
[   11.040805] pci_hotplug: PCI Hot Plug PCI Core version: 0.5
[   11.043188] pciehp: PCI Express Hot Plug Controller Driver version: 0.4
[   11.046201] input: Power Button as /devices/LNXSYSTM:00/LNXPWRBN:00/input/input0
[   11.049319] ACPI: Power Button [PWRF]
[   11.050923] input: Sleep Button as /devices/LNXSYSTM:00/LNXSLPBN:00/input/input1
[   11.053989] ACPI: Sleep Button [SLPF]
[   11.055822] GHES: HEST is not enabled!
[   11.057973] xen:grant_table: Grant tables using version 1 layout
[   11.060557] Grant table initialized
[   11.062129] Cannot get hvm parameter CONSOLE_EVTCHN (18): -22!
[   11.064628] Serial: 8250/16550 driver, 32 ports, IRQ sharing enabled
[   11.093519] 00:06: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[   11.099381] Linux agpgart interface v0.103
[   11.103654] loop: module loaded
[   11.105116] Invalid max_queues (4), will use default max: 2.
[   11.109795] scsi host0: ata_piix
[   11.111415] scsi host1: ata_piix
[   11.113157] ata1: PATA max MWDMA2 cmd 0x1f0 ctl 0x3f6 bmdma 0xc100 irq 14
[   11.116759] ata2: PATA max MWDMA2 cmd 0x170 ctl 0x376 bmdma 0xc108 irq 15
[   11.117846] blkfront: xvda: barrier or flush: disabled; persistent grants: disabled; indirect descriptors: enabled;
[   11.119786]  xvda: xvda1
[   11.122940] blkfront: xvdb: flush diskcache: enabled; persistent grants: disabled; indirect descriptors: enabled;
[   11.131551] libphy: Fixed MDIO Bus: probed
[   11.134438] tun: Universal TUN/TAP device driver, 1.6
[   11.136597] tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
[   11.139377] PPP generic driver version 2.4.2
[   11.141256] VMware vmxnet3 virtual NIC driver - version 1.4.a.0-k-NAPI
[   11.144112] xen_netfront: Initialising Xen virtual ethernet driver
[   11.146699] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[   11.149409] ehci-pci: EHCI PCI platform driver
[   11.151277] ehci-platform: EHCI generic platform driver
[   11.153522] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[   11.156153] ohci-pci: OHCI PCI platform driver
[   11.158042] ohci-platform: OHCI generic platform driver
[   11.160238] uhci_hcd: USB Universal Host Controller Interface driver
[   11.163021] i8042: PNP: PS/2 Controller [PNP0303:PS2K,PNP0f13:PS2M] at 0x60,0x64 irq 1,12
[   11.168231] serio: i8042 KBD port at 0x60,0x64 irq 1
[   11.170519] serio: i8042 AUX port at 0x60,0x64 irq 12
[   11.172888] mousedev: PS/2 mouse device common for all mice
[   11.176081] input: AT Translated Set 2 keyboard as /devices/platform/i8042/serio0/input/input2
[   11.176267] rtc_cmos 00:02: rtc core: registered rtc_cmos as rtc0
[   11.176346] rtc_cmos 00:02: alarms up to one day, 114 bytes nvram, hpet irqs
[   11.176353] i2c /dev entries driver
[   11.176421] device-mapper: uevent: version 1.0.3
[   11.176504] device-mapper: ioctl: 4.35.0-ioctl (2016-06-23) initialised: dm-devel@redhat.com
[   11.192425] ledtrig-cpu: registered to indicate activity on CPUs
[   11.195814] NET: Registered protocol family 10
[   11.198010] NET: Registered protocol family 17
[   11.199887] 9pnet: Installing 9P2000 support
[   11.201742] Key type dns_resolver registered
[   11.203815] divide error: 0000 [#1] SMP
[   11.205591] Modules linked in:
[   11.206929] CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.9.80-rancher #1
[   11.207711] Hardware name: Xen HVM domU, BIOS 4.2.amazon 08/24/2006
[   11.207711] task: ffff8f8b0eac8000 task.stack: ffffb64f018a8000
[   11.207711] RIP: 0010:[<ffffffffb3da00b4>]  [<ffffffffb3da00b4>] init_intel_microcode+0x4b/0x5c
[   11.207711] RSP: 0000:ffffb64f018abe20  EFLAGS: 00010206
[   11.207711] RAX: 0000000001900000 RBX: 00000000ffffffea RCX: 0000000000000000
[   11.207711] RDX: 0000000000000000 RSI: 0000000000000006 RDI: ffffffffb3d9fc67
[   11.207711] RBP: ffffb64f018abe80 R08: fffffffffffffffc R09: 0000000000000003
[   11.207711] R10: 0000000000000000 R11: ffff8f8b0d9da184 R12: 00000000ffffffff
[   11.207711] R13: 0000000000000000 R14: 0000000000000008 R15: ffffffffb3d8d842
[   11.207711] FS:  0000000000000000(0000) GS:ffff8f8b20240000(0000) knlGS:0000000000000000
[   11.207711] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
[   11.207711] CR2: ffffb64f019ac000 CR3: 0000000240c08000 CR4: 0000000000160670
[   11.207711] Stack:
[   11.207711]  ffffffffb3d9fc91 0000000100000002 ffffffffb3297cbe 0000000000000001
[   11.207711]  ffffffffb3d65a00 0000000000000000 0000000000000008 ffffffffb3339ac6
[   11.207711]  ffffb64f018abe80 ffffffffb3d9e2a1 59e5b08c9c14e482 ffffffffb3d9fc67
[   11.207711] Call Trace:
[   11.207711]  [<ffffffffb3d9fc91>] ? microcode_init+0x2a/0x1c6
[   11.207711]  [<ffffffffb3297cbe>] ? __debugfs_create_file+0xd1/0xe3
[   11.207711]  [<ffffffffb3339ac6>] ? find_next_bit+0x18/0x1b
[   11.207711]  [<ffffffffb3d9e2a1>] ? threshold_init_device+0x24/0x4d
[   11.207711]  [<ffffffffb3d9fc67>] ? save_microcode_in_initrd+0x45/0x45
[   11.207711]  [<ffffffffb30021f1>] do_one_initcall+0xa5/0x143
[   11.207711]  [<ffffffffb3d8e0e9>] kernel_init_freeable+0x184/0x212
[   11.207711]  [<ffffffffb36d9e42>] ? rest_init+0x79/0x79
[   11.207711]  [<ffffffffb36d9e50>] kernel_init+0xe/0xf0
[   11.207711]  [<ffffffffb36e41a7>] ret_from_fork+0x57/0x70
[   11.207711] Code: 1e 73 14 55 48 c7 c7 ed 3b a8 b3 48 89 e5 e8 3a 65 3a ff 31 c0 5d c3 8b 05 aa 40 fc ff 0f b7 0d c3 40 fc ff 31 d2 c1 e0 0a 48 98 <48> f7 f1 89 05 03 bb 15 00 48 c7 c0 60 d9 c2 b3 c3 4c 8d 54 24 
[   11.207711] RIP  [<ffffffffb3da00b4>] init_intel_microcode+0x4b/0x5c
[   11.207711]  RSP <ffffb64f018abe20>
[   11.296397] ---[ end trace aa79dc13cc53375f ]---
[   11.299151] Kernel panic - not syncing: Fatal exception
[   11.300366] Kernel Offset: 0x32000000 from 0xffffffff81000000 (relocation range: 0xffffffff80000000-0xffffffffbfffffff)
[    0.000000] Linux version 4.9.80-rancher (root@d8d803e7a6cd) (gcc version 7.3.0 (GCC) ) #1 SMP Tue Mar 27 02:21:37 UTC 2018
[    0.000000] Command line: BOOT_IMAGE=../vmlinuz-4.9.80-rancher printk.devkmsg=on rancher.state.dev=LABEL=RANCHER_STATE rancher.state.wait panic=10 console=ttyS0 rancher.cloud_init.datasources=["ec2"] initrd=../initrd-v1.3.0-rc2
[    0.000000] KERNEL supported cpus:
[    0.000000]   Intel GenuineIntel
[    0.000000]   AMD AuthenticAMD
[    0.000000]   Centaur CentaurHauls
[    0.000000] x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
[    0.000000] x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
[    0.000000] x86/fpu: Enabled xstate features 0x7, context size is 832 bytes, using 'standard' format.
[    0.000000] x86/fpu: Using 'eager' FPU context switches.
[    0.000000] e820: BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009dfff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009e000-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000e0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x00000000efffffff] usable
[    0.000000] BIOS-e820: [mem 0x00000000fc000000-0x00000000ffffffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x00000003e08fffff] usable
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.7 present.
[    0.000000] Hypervisor detected: Xen
[    0.000000] Xen version 4.2.
[    0.000000] Netfront and the Xen platform PCI driver have been compiled for this kernel: unplug emulated NICs.
[    0.000000] Blkfront and the Xen platform PCI driver have been compiled for this kernel: unplug emulated disks.
[    0.000000] You might have to change the root device
[    0.000000] from /dev/hd[a-d] to /dev/xvd[a-d]\
```

**Comment by niusmallnan** [comment]
I tried r4 and it worked fine. The r4 is the current generation.
Keep this open.
If anybody needs to use r3, please let me know.

**Comment by sijmenvos** [comment]
This also happens on m3.large

**Comment by kingsd041** [comment]
@sijmenvos  The m3.large and r3.large instance types are already available in rancheros v1.4.0-rc2
